content/en/containers/kubernetes/data_collected.md

```diff
@@ -25,23 +25,31 @@ further_reading:
   text: "Assign tags to all data emitted by a container"
 ---
 
-## Metrics
+This page lists data collected by the Datadog Agent when deployed on a Kubernetes cluster. 
 
-Metrics collected by the Agent when deployed on your Kubernetes cluster:
+The set of metrics collected may vary depending on the version of Kubernetes in use.
 
-**Note**: The set of metrics collected by the Datadog Kubernetes integration may vary depending on the version of Kubernetes in use.
+## Metrics
 
 ### Kubernetes
 
 {{< get-metrics-from-git "kubernetes" >}}
 
 ### Kubelet
 
+For more information, see the documentation for the [Kubelet][1] integration.
+
 {{< get-metrics-from-git "kubelet" >}}
 
+### Kubernetes state metrics core
+
+For more information, see the documentation for the [Kubernetes state metrics core][6] integration. This check requires Datadog Cluster Agent v1.12+.
+
+{{< get-metrics-from-git "kubernetes_state_core" >}}
+
 ### Kubernetes state
 
-Note that `kubernetes_state.*` metrics are gathered from the `kube-state-metrics` API.
+Note that `kubernetes_state.*` metrics are gathered from the `kube-state-metrics` API. The `kubernetes_state` check is a legacy check. For an alternative, see [Kubernetes state metrics core][6]. Datadog recommends that you do not enable both checks simultaneously.
```

Comment:
`Note that` is listed on an internal resource of phrases to avoid, which I'm happy to share via Slack if you'd like. I'm also happy to continue iterating on this if the **Note** doesn't seem like a good fit to you.